data/mods/MindOverMatter/obsolete/upgrade_eocs/electrokinetic_upgrades.json

```diff
@@ -41,6 +42,14 @@
     },
     "effect": [ { "u_forget_recipe": "practice_electrokinetic_melee_attacks" } ]
   },
+  {
```

Comment:
This part shouldn't be necessary, since there are no existing characters with this power that need to be upgraded to the power learning system